package.json

```diff
@@ -81,14 +81,14 @@
     "lint-staged": "^15.0.1",
     "pinst": "^3.0.0",
     "prettier": "^3.0.3",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
     "rollup-plugin-preserve-directives": "^0.2.0",
     "typescript": "^5.2.2"
   },
   "peerDependencies": {
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0"
```

Comment:
How about `^18.0.0 || ^19.0.0`? otherwise dropping react 18 would require a major release, I'd prefer to avoid that